modules/store/spec/modules.spec.ts

```diff
@@ -216,4 +216,34 @@ describe(`Store Modules`, () => {
       });
     });
   });
+
+  describe(`: Slice-Like paramter`, () => {
```

Comment:
Should it be tested with the real RTK slice as an integration spec?

Reply:
I don't think that that is needed.

Reply:
I agree

Reply:
I agree

Reply:
That being said, perhaps we could add a recipe to use NgRx with RTK in the future, as a separate PR.